config.ru

```diff
@@ -1,3 +1,4 @@
+$stdout.sync = true
```

Comment:
yeah, i was thinking we probably should. it's been sort of a pain debugging some of the slack-specific stuff, this should make it easier